report-api/report-templates/statement_report.html

```diff
@@ -1,405 +1,414 @@
+      width: 50%;
+    }
+
+    .section-title {
+      margin: 10px 24px 0px;
+      padding-bottom: 16px;
+      font-family: 'BCSans-Bold', sans-serif;
+      font-size: 18px;
+      color: #234075;
+    }
+
+    .no-border {
+      border: none;
+      padding: 12px 0 0 0;
+    }
+
+    .mb-0 {
+      margin-bottom: 0px;
+    }
+
+    .transaction-table {
+      margin: 0px 24px 16px;
+      width: 690px;
+    }
+
+    .transaction-table thead tr,
+    .transaction-table tbody tr {
+      line-height: 3;
+      font-size: 12px;
+      border-bottom: 1px solid #bcbec5;
+    }
+
+    .transaction-table thead tr,
+    .transaction-table tbody tr.row-border {
+      border-bottom: 1px solid #b3bac3;
+    }
+
+    .transaction-table thead th {
+      text-align: left;
+      vertical-align: top;
+      font-family: 'BCSans-Bold', sans-serif;
+      font-weight: bold;
+      padding-bottom: 2px;
+    }
+    .transaction-table thead th:last-child {
+      text-align: right;
+    }
+    .transaction-table tbody td:last-child {
+      text-align: right;
+    }
+
+    .transaction-table tbody .product-purchased {
+      line-height: 1.5;
+      padding-top: 10px;
+      padding-bottom: 10px;
+      max-width: 135px;
+    }
+
+    .transaction-table thead .two-line-head {
+      line-height: 1.5;
+      padding-top: 10px;
+      padding-bottom: 10px;
+    }
+
+    .total-elements {
+      margin: 0px 24px;
+      padding: 10px 0;
+      font-size: 14px;
+      border-bottom: 1px solid #b3bac3;
+    }
+
+    .total-elements-price {
+      float: right;
+      font-family: 'BCSans-Regular', sans-serif;
+    }
+
+    .statement-footer {
+      padding: 10px 24px 20px;
+      position: fixed;
+      bottom: 0px;
+      left: -24px;
+      width: 100%;
+      margin-bottom: -130px;
+    }
+
+    .footer-bcrs-logo {
+      display: inline-block;
+      padding-bottom: 16px;
+      padding-left: 16px;
+      float: left !important;
+    }
+
+    .footer-bcrs-logo img {
+      width: 180px;
+    }
+
+    .statement-footer .footer-info {
+      font-size: 13px;
+      display: inline-block;
+      vertical-align: middle;
+      text-align: right !important;
+      float: right;
+      padding-right: 34px;
+      margin-top: 30px;
+      color: #234075;
+    }
+
+    .statement-footer .footer-info span {
+      padding: 2px 7px;
+      border-right: 1px solid #868686;
+      display: inline-block;
+      text-align: right;
+      margin-right: 3px;
+    }
+
+    .statement-footer .footer-info span:last-child {
+      border-right: none;
+      color: #234075;
+    }
+
+    .font-bold {
+      font-family: 'BCSans-Bold', sans-serif;
+      font-weight: bold;
+    }
+
+    .page-break-before {
+      page-break-before: always;
+    }
+
+    .no-margin {
+      margin-top: 0!important;
+    }
+
+  </style>
+</head>
+<body>
+  {% set payment_methods = ["EFT", "PAD", "CC", "DRAWDOWN", "INTERNAL", "DIRECT_PAY", "WIRE", "ONLINE_BANKING", "EJV", "CASH", "CHEQUE"] %}
```

Comment:
Cash and cheque are the same payment method (Routing Slip)
I don't think there is a WIRE payment method